### PR TITLE
NPM Script Update & Readme

### PR DESCRIPTION
**Pull request by cdcabrera**
## Description

Goal, simplify install process for users unfamiliar with dev installs. Implementation of NPM scripts as aliases/shortcuts towards Grunt tasks and automated Jekyll postinstall. 

May need some review input on the "jekyll" NPM script within package.json to verify syntax. Per @bleathem removed redundant jekyll install.

## Todos
- N/A, cross browser test
- N/A, Are you sure it works on IE9?
- N/A, Is it reponsive?

## Steps to test or reproduce and link to rawgit

```sh
$ git pull https://github.com/cdcabrera/patternfly.git NPM-Scripts-4x:NPM-Scripts-4x
$ npm install
$ npm start
```

@bleathem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,8 +101,15 @@ Additionally you may need to install the grunt command line utility.  To do this
...
-    gem install bundle
-    bundle install
+```
+npm run jekyll
+```
+
+or
+
+```
+gem install jekyll -v 3.1.6
```
Comment (bleathem) [inline README.md]: The preferred way to install jekyll is using `bundle install` so we all have the same dependency versions.

Comment (bleathem) [reply README.md]: So +1 for the `npm run`wrapper, but the alternate command should be to use `bundle install`

**Review by bleathem — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cdcabrera) [reply README.md]: Update applied. Looks like the rebase messed around with the review comment... should now appear as
```
gem install bundle
bundle install
```

File: package.json [hunk 2]
```diff
@@ -65,7 +66,11 @@
     "patternfly-bootstrap-treeview": "~2.1.0"
   },
   "scripts": {
-    "test": "./node_modules/karma/bin/karma start"
+    "test": "grunt karma",
+    "build": "grunt build",
+    "jekyll": "ruby --version || echo \"Ruby required\"; gem install bundler || sudo gem install bundler; bundle install; set -e",
```
Comment (bleathem) [inline package.json]: Let's drop the sudo part here

**Review by bleathem — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (cdcabrera) [reply package.json]: updated

**Review by bleathem — APPROVED** [review]

**Pull request merged**